doc/cla/individual/flh.md

```diff
@@ -0,0 +1,11 @@
+France, 2015-08-18
+
+I hereby agree to the terms of the Odoo Individual Contributor License
+Agreement v1.0.
+
+I declare that I am authorized and able to make this agreement and sign this
+declaration.
+
+Signed,
+
+Florian Hatat github@hatat.me https://github.com/flh
```

Comment:
you need to sign the CLA with the same email as the one you are using to commit (in your case the `@free.fr` address)